scripts/build.sh

```diff
@@ -272,7 +272,7 @@ case "$product-$method-$platform" in
 
     echo "Building cmake build ..."
     cpus=$(sysctl -n hw.ncpu)
-    (cd build; env make -j $cpus all)
+    (cd build; env make -j $cpus all generate_protos)
```

Comment:
Don't you want these in the other order so that make all runs against the generated protos?

Reply:
It doesn't matter. We've got it wired such that make will generate the nanopb/cpp protos during the main ('all') phase of the build and use them. The 'generate_protos' target isn't necessary at all. Except then the objc protos won't actually be built, since they're not used during the cmake build.

We could leave it off altogether, except then we wouldn't notice when the objc protos should be rebuilt (hence why objc proto changes appear in this PR.)